test/index.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai';
+import mockFs from 'mock-fs';
+import { useFakeTimers, SinonFakeTimers } from 'sinon';
+import { promises as fs } from 'fs';
+
+import blinking from '../index';
+
+describe('blinking', () => {
+    const LED_FILE_PATH = '/sys/class/leds/led0/brightness';
+    let clock: SinonFakeTimers;
+    const blink = blinking(LED_FILE_PATH);
+
+    const expectValue = async (path: string, expected: string) =>
+        expect(await fs.readFile(path, 'utf-8')).to.equal(expected);
+    const expectOn = async () => await expectValue(LED_FILE_PATH, '1');
+    const expectOff = async () => await expectValue(LED_FILE_PATH, '0');
```

Comment:
The `On` `Off` suffixes are usually associated with events (at least for me), maybe we can be a bit more explicit for better readability? Something like

```suggestion
    const expectLedTurnOn = async () => await expectValue(LED_FILE_PATH, '1');
    const expectLedTurnOff = async () => await expectValue(LED_FILE_PATH, '0');
```

Reply:
Sure!

Reply:
Changed in https://github.com/balena-io-modules/blinking/commit/d15e090f8d0af06c973e558eed32a2854fd70f06